index.html

```diff
@@ -0,0 +1,182 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link rel="stylesheet" href="css/base.css">
+    <link rel="stylesheet" href="css/layout.css">
+    <link rel="stylesheet" href="css/module.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+    <header class="containerTopMargin">
+      <div class= "container">
+        <div class="row">
+          <div class="six columns">
+            <img src="images/logo.png" alt="logo" class="logo leftFloat">
+            <section class="logo">
+              <h1 class="logo__logoStyleOne" class="logo__logoText rightFloat">Delicious</h1>
+              <h2 class="logo__logoStyleTwo" class="logo__logoText rightFloat">THE BEST FOOD BLOG ON THE WEB.</h2>
+            </section>
+          </div>
+          <div class="six columns">
+            <nav>
+              <ul>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/fb-icon.png" alt="facebook">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/twit-icon.png" alt="twitter">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/gp-icon.png" alt="google plus">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/insta-icon.png" alt="instagram">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/flic-icon.png" alt="flickr">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/pint-icon.png" alt="pintrest">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/rss-icon.png" alt="rss">
+                  </a>
+                </li>
+                <li class="nav rightFloat">
+                  <a href="">
+                    <img src="images/mail-icon.png" alt="mail">
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <article class="containerTopMargin borderTop">
+        <section>
+          <div class="container">
+            <div class="row">
+              <div class="six columns">
+                <h2 class="recipeHeader">Chocolate Pizza</h2>
+                <h3 class="postDate textGray">POSTED ON 15 DEC 2013<time datetime="2013-12-13" pubdat>/DESSERTS</h3>
```

Comment:
use css `text-transform: uppercase` for things like this. If the allcaps don't have any meaning, i.e. not an acronym or something similar, this is trying to add style in your markup and is generally bad practice.